### ISSUE TITLE
Make "expand all" button

### ISSUE THREAD

**Issue by julie-sullivan** [MICommunity/interaction-viewer#79]
From Gos

```
I realise it would be nice to toggle *all* the nodes from circle to rectangle (so can see
where they interact) simultaneously.  Maybe they could be done invididually as now 
but the Key of the diagram could all toggle of all?
```

**Comment by colin-combe** [comment]
Yes - easily done.
Incidentally, see  https://github.com/colin-combe/interaction-viewer/issues/1 ;)

**Comment by colin-combe** [comment]
fixed by commit cd4b6ec 

call either .expandAll() or .collapseAll() on your instance of the interaction viewer Controller. E.g. see:
https://github.com/MICommunity/interaction-viewer/blob/v1_bug_fixes/index.html#L41-L42

**Comment by bmeldal** [comment]
Moved to our tracker to commit changes in CP.
